.cm/ask-ai.cm

```diff
@@ -12,6 +12,8 @@ automations:
     if:
       - true
     run:
-      - action: add-comment@v1
+      - action: update-description@v1
         args:
-          comment: "{{ source | askAI("based on the given context, describe in plain english the changes introduced in this PR.", env.OPEN_AI_TOKEN) | prependTitle("gitStream Describe PR") | encode }}"
\ No newline at end of file
+          concat_mode: append
+          description: |
+            "{{ source | askAI("based on the given context, describe in plain english the changes introduced in this PR.", env.OPEN_AI_TOKEN) | prependTitle("✨ /:\ gitStream Describe PR ✨") | encode }}"
```

Comment:
I would add a line before saying something like `_the following was added by gitStream askAI plugin_`

Reply:
you say it is done automatically? I dont think so. Or do you mention it as a reference for such behavior?

Reply:
It's done by piping the output of the filter to `prependTitle` (another plugin that adds a prefix markdown title)